kroxylicious/src/main/java/io/kroxylicious/proxy/internal/util/Metrics.java

```diff
@@ -6,22 +6,55 @@
 
 package io.kroxylicious.proxy.internal.util;
 
+import java.util.List;
+
+import org.apache.kafka.common.protocol.ApiKeys;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.DistributionSummary;
 import io.micrometer.core.instrument.Tag;
 
+import static io.micrometer.core.instrument.Metrics.counter;
+import static io.micrometer.core.instrument.Metrics.summary;
+
 public class Metrics {
 
     // creating a constant for all Metrics in the one place so we can easily see what metrics there are
 
-    public static final String KROXYLICIOUS_INBOUND_DOWNSTREAM_MESSAGES = "kroxylicious_inbound_downstream_messages";
+    private static final String KROXYLICIOUS_INBOUND_DOWNSTREAM_MESSAGES = "kroxylicious_inbound_downstream_messages";
+
+    private static final String KROXYLICIOUS_INBOUND_DOWNSTREAM_DECODED_MESSAGES = "kroxylicious_inbound_downstream_decoded_messages";
+
+    private static final String KROXYLICIOUS_REQUEST_SIZE_BYTES = "kroxylicious_request_size_bytes";
```

Comment:
As we use this counter for requests and responses (discriminated by flowing), would rpc_payload_size be a better name?

Reply:
Any thoughts on the naming?

Reply:
`rpc_payload_size` is definitely better, but I wonder if the `rpc_` prefix is redundant? Maybe just plain `payload_size`. 

Note: If we are renaming this we should note it as a breaking change in the public API in the release notes (I don't think we need to do more than that at this stage)